sqlmesh/core/model/definition.py

```diff
@@ -881,6 +882,7 @@ def _data_hash_values(self) -> t.List[str]:
             self.physical_version,
             self.gateway,
             self.interval_unit.value if self.interval_unit is not None else None,
+            self.optimize,
```

Comment:
This needs to be converted into a string right? The items in this list are supposed to be strings, why does it work if this is a bool?

Reply:
Right, on the metadata hash there was an explicit error regarding the conversion, here it wasn't apparent. Will switch to `str(self.optimize)`